modules/cats/shared/src/main/scala/eu/timepit/refined/cats/syntax.scala

```diff
@@ -23,5 +23,15 @@ trait CatsRefinedTypeOpsSyntax {
 trait CatsNonEmptyListSyntax {
   implicit class CatsNonEmptyListRefinedOps[A](nel: NonEmptyList[A]) {
     def refinedSize: PosInt = PosInt.unsafeFrom(nel.size)
+
+    def take(n: PosInt): NonEmptyList[A] =
+      NonEmptyList.fromListUnsafe(nel.toList.take(n.value))
```

Comment:
This could be made a bit more performant by not calling `toList`:
```suggestion
      NonEmptyList(nel.head, nel.tail.take(n.value - 1))
```

Reply:
Thank you for the comment!
I committed this suggestion.